StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp11/DocumentationRules/SA1600CSharp11UnitTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#nullable disable
+
+namespace StyleCop.Analyzers.Test.CSharp11.DocumentationRules
+{
+    using Microsoft.CodeAnalysis.Testing;
+    using StyleCop.Analyzers.Test.CSharp10.DocumentationRules;
+    using static StyleCop.Analyzers.Test.Verifiers.StyleCopCodeFixVerifier<
+        StyleCop.Analyzers.DocumentationRules.SA1600ElementsMustBeDocumented,
+        StyleCop.Analyzers.DocumentationRules.SA1600CodeFixProvider>;
+
+    public class SA1600CSharp11UnitTests : SA1600CSharp10UnitTests
+    {
+        protected override DiagnosticResult[] GetExpectedResultTestRegressionMethodGlobalNamespace(string code)
```

Comment:
❔ Is this different from the base class?

Reply:
I believe this override can be removed and the C# 11 test just use the one from C# 10

Reply:
:thumbsup:

Reply:
:thumbsup: